allennlp/models/encoder_decoders/wikitables_semantic_parser.py

```diff
@@ -269,6 +339,57 @@ def forward(self,  # type: ignore
             # TODO(matt): compute accuracy here.
             return outputs
 
+    def _compute_average_with_encoder(self, embed_tensor: Variable,
+                                      mask: Variable) -> Variable:
+        # computes the average along dimension 1 of the embed_tensor
+        dims = embed_tensor.size()
+        table_mask = mask.view(-1, mask.size(2))
+        averaged_tensor = self._entity_encoder(embed_tensor.view(-1, dims[2], dims[3]),
+                                               table_mask)
+        # (batch_size, num_entities, embedding_dim)
+        return averaged_tensor.view(dims[0], dims[1], dims[3])
+
+    def _get_neighbor_indexes(self, world: List[WikiTablesWorld],
```

Comment:
Indentation is off here - `world` should be on the next line, and `max_num_entities` should be aligned with `self` and `world`.